test/box/net.box_incompatible_index-gh-1729.test.lua

```diff
@@ -26,8 +26,3 @@ c:eval('return echo(...)', {42})
 c:call('echo', 42)
 c:eval('return echo(...)', 42)
 c:close()
-c = net.connect(box.cfg.listen, {call_16 = true})
-c:call('echo', 42)
-c:eval('return echo(...)', 42)
-c:close()
-box.schema.user.revoke('guest', 'execute', 'universe')
```

Comment:
Nit: You need to drop more than that - up to `-- CALL vs CALL_16 in connect options`.

Reply:
Why should I drop the part testing regular `call`?

Reply:
Because otherwise you drop half of a test case named "CALL vs CALL_16". This makes the test case contradict its title. We should either fix the test case somehow so that it makes sense or drop it. I'd drop it, because there should be another test case checking `call` arguments somewhere.

Reply:
Applied your suggestion.